.travis.yml

```
@@ -45,13 +45,19 @@ matrix:
     <<: *OPAM
   - env:
     - COQ_IMAGE=coqorg/coq:8.10
+    - CONTRIB_NAME=coq-toychain-node
+    - NJOBS=2
+    - SWITCH=4.07.1+flambda
+    <<: *OPAM
+  - env:
+    - COQ_IMAGE=coqorg/coq:8.11
```

If speed is an issue, we can use the MathComp image to save some time:
```suggestion
    - COQ_IMAGE=mathcomp/mathcomp:1.10.0-coq-8.11
```